src/CloudFlare/IpRewrite.php

```diff
@@ -69,38 +82,32 @@ public static function reset()
     protected static function rewrite()
     {
         // only should be run once per page load
-        if (self::$is_cf !== NULL) {
+        if (self::$is_loaded) {
```

Comment:
Can you just check if `self::$rewritten_ip === null` if it is then you know you haven't loaded yet right? Then we can remove the `$is_loaded`.  Maybe I'm missing something